lib/cluster/index.js

```diff
@@ -453,15 +454,28 @@ Cluster.prototype.sendCommand = function (command, stream, node) {
         if (!random) {
           if (typeof targetSlot === 'number' && _this.slots[targetSlot]) {
             var nodeKeys = _this.slots[targetSlot];
-            var key;
-            if (to === 'all') {
-              key = utils.sample(nodeKeys);
-            } else if (to === 'slave' && nodeKeys.length > 1) {
-              key = utils.sample(nodeKeys, 1);
+            if (typeof to === 'function') {
+              var nodes =
+                  nodeKeys
+                  .slice(1)
+                  .map(function(key) {
+                    return _this.connectionPool.nodes.all[key];
+                  });
+              redis = to(nodes, command);
+              if (redis.constructor === Array) {
```

Comment:
As a hint: there's a easy [polyfill](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Polyfill) for this and Microsoft itself does [not maintain IE 8](https://support.microsoft.com/en-us/lifecycle#gp/Microsoft-Internet-Explorer) or older, not even with security fixes on any regular Windows version. Most libraries drop support for it or have already dropped the support and the faster all developers switch to not supporting it anymore, the faster no one is going to visit any website with it anymore.